### ISSUE TITLE
`rule.error-strings` does not catch `errors.Wrap` messages

### ISSUE THREAD

**Issue by mrd0ll4r** [mgechev/revive#608]
Using `rule.error-strings`, I expected a warning from code like this:
```
err := somethingFallible()
if err != nil {
  return errors.Wrap(err, "Something terrible happened.")
}
```

I'm not sure if the usual error string formatting applies here, but I would expect that it does...

 - OS: Latest Ubuntu (e.g. whatever is used in GitHub runners)
 - Version of Go: 1.16
